pom.xml

```diff
@@ -606,7 +635,7 @@
           <webApp>
               <contextPath>/jenkins</contextPath>
           </webApp>
-          <minimumJavaVersion>${plugin.minimumJavaVersion}</minimumJavaVersion>
+          <minimumJavaVersion>1.${java.level}</minimumJavaVersion>
```

Comment:
I think the reason to make this a property was so that you could override it and set 11. With it here the only way to do that is to override the plugin configuration

Reply:
I don't se how. If you require 11 at run time, then you should also compile with java 11..

compiling with a different java version that you require at run-time is highly likely to make things go bang at run-time - its such an advanced use case I do not believe we should hand a loaded gun to non advanced people so they can shoot themselves in the foot.  Either that or I am missing a use case.

Reply:
I'm not actually sure what the state of Java 11 is here. #133 is still open indicating that the only option today should be java 8.  (I still to remove Java7 support to simplify)